packages/compass/src/app/components/home.tsx

```diff
@@ -278,26 +281,28 @@ function Home({
 
   const connectionStorage =
     __TEST_CONNECTION_STORAGE === undefined
-      ? ConnectionStorage
+      ? getCompassRendererConnectionStorage()
```

Comment:
See my comment above about maybe changing the places where this method is used to actual hooks that return the service, we can just construct an instance here then if we do this